torchx/schedulers/ray/ray_driver.py

```diff
@@ -95,7 +95,12 @@ def schedule(self, actor_id: str) -> CommandActorScheduled:
         return CommandActorScheduled(actor_id)
 
     def get_actor_address_and_port(self) -> Tuple[str, int]:
-        return get_address_and_port()
+        addr = ray.util.get_node_ip_address()
+        with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
```

Comment:
Did you mean to use https://docs.python.org/3/library/contextlib.html#contextlib.closing  here?